models/idea/index.js

```diff
@@ -274,6 +274,40 @@ class Idea extends Model {
     const cursor = await this.db.query(query, params);
     return await cursor.all();
   }
+
+
+  /**
+   * Read ideas commented by specified users
+   * @param {string[]} minimum rate - minimal query rate
+   * @param {integer} offset - pagination offset
+   * @param {integer} limit - pagination limit
+   * @returns {Promise<Idea[]>} - list of found ideas
+   */
+  static async findHighlyRated(rateBottomLimit, { offset, limit }) {
+    const query = `
+      FOR idea IN ideas
+        FOR vote IN votes
+        FILTER idea._id == vote._to
+        // group by idea id
+        COLLECT id = idea
+        // get sum of each idea's votes values
+        AGGREGATE rate = SUM(vote.value)
+        // set bottom limit of rate
+        FILTER rate >= @rateBottomLimit
```

Comment:
We sort by rate but we may not be willing to show ideas with 0 or lower than zero rates.

Reply:
May we actually want to restrict people from seeing low voted ideas? Why?

Well, maybe it doesn't matter. I might be just bikeshedding.

Might be a cliche, but: _we should make things as simple as possible, but not simpler_.

Reply:
I think it is sometimes not necessary to discuss things like this and just let them be if it is really one documented variable which if clear what is it for.
And dry for being dry or simple for being simple, not for saving time and energy for a different work.
Is there any case this variable can be harmful?

Reply:
Yes. I totally understand your point @agatatalita. I'm nitpicking here.
I don't understand the strong resistance though.

I believe if something can be simplified, it should be. I'm willing to make the changes myself, only if I get approval for them.

Unless we leak some information, the read-only can't be harmful.
The only meaningful value is 0, unless we want to add a user interface for this, which I doubt. (Or do we?) And then it limits our ability to see negatively voted ideas in this list.

If you're still not convinced, I'm leaving this discussion and we keep it as it is.

Reply:
I gave a few examples when it can be useful. Why would I be writing it again later if I was able to put the parameter now or that was my imagination of the use case? 

Generally also the name highlyVotedIdeas can be confusing. Because we are actually listing all of the ideas, not just highly voted.

Reply:
> the name highlyVotedIdeas can be confusing

This is true, actually. I haven't thought about this. Maybe it should be `?sort=-voteSum` or similar. Then we could also have `?sort=voteSum` to find the absolutely worst ideas. :)
Maybe also random should be `?sort=random`.
Not sure if this matters. :)

Reply:
I believe all of this filtering and sorting will be rewritten at some point (because it is quite bothering to be not able to connect sort with filters and I believe people may expect this - so there will be one filtering sorting function or some function for creating queries probably?).

Reply:
@agatatalita I apologize for the fuss around this. I realize it really is a minor thing. It just takes time for it to fit my rigid mindset. :)

I don't know what future holds, but indeed some nice generalizing refactor may be beneficial at some point.